app/views/email_alert_subscriptions/facets/checkbox.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render "govuk_publishing_components/components/checkboxes", {
+    name: nil,
+    heading: @signup_presenter.name,
+    hint_text: "Select the email alerts you need.",
+    items: @signup_presenter.choices_formatted,
+    is_page_heading: true,
+} %>
```

Comment:
I see the text "Select the email alerts you need." on the email subscription page, but no checkbox. Is this component used just for the heading?